tsinfer/formats.py

```diff
@@ -1419,8 +1419,8 @@ def min_site_times(self, individuals_only=False):
     def from_tree_sequence(
         cls,
         ts,
-        use_sites_time=True,
-        use_individuals_time=True,
+        use_sites_time=False,
```

Comment:
Let's change these to None while we're breaking things. We may want to have different behaviour sometime in the future and hard-coding the defaults in the signature makes things very inflexible.

Reply:
Good call.